packages/esm-patient-medications-app/src/drug-order-basket-panel/drug-order-basket-panel.extension.tsx

```diff
@@ -19,10 +19,41 @@ export default function DrugOrderBasketPanelExtension() {
   const isTablet = useLayoutType() === 'tablet';
   const { orders, setOrders } = useOrderBasket<DrugOrderBasketItem>('medications', prepMedicationOrderPostData);
   const [isExpanded, setIsExpanded] = useState(orders.length > 0);
-  const newOrderBasketItems = orders.filter((x) => x.action === 'NEW');
-  const renewedOrderBasketItems = orders.filter((x) => x.action === 'RENEW');
-  const revisedOrderBasketItems = orders.filter((x) => x.action === 'REVISE');
-  const discontinuedOrderBasketItems = orders.filter((x) => x.action === 'DISCONTINUE');
+  const {
+    incompleteOrderBasketItems,
+    newOrderBasketItems,
+    renewedOrderBasketItems,
+    revisedOrderBasketItems,
+    discontinuedOrderBasketItems,
+  } = useMemo(() => {
+    const incompleteOrderBasketItems: Array<DrugOrderBasketItem> = [];
+    const newOrderBasketItems: Array<DrugOrderBasketItem> = [];
+    const renewedOrderBasketItems: Array<DrugOrderBasketItem> = [];
+    const revisedOrderBasketItems: Array<DrugOrderBasketItem> = [];
+    const discontinuedOrderBasketItems: Array<DrugOrderBasketItem> = [];
+
+    orders.forEach((order) => {
+      if (order?.isOrderIncomplete) {
+        incompleteOrderBasketItems.push(order);
```

Comment:
Q: Should incomplete items be ordered predictably by action? This is probably fine for now.

Reply:
My thought here was to show all incomplete orders at the top.

Reply:
Oh, that completely makes sense and I get that. I was more of wondering if the incomplete orders themselves should be sorted like the other orders while being on top.